Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:14.18.0-stretch-slim
+LABEL maintainer="Andrey Ivanov (https://github.com/343dev)"
+
+WORKDIR /app
+
+COPY . .
+
+ENV NODE_ENV="production"
+
+RUN apt update \
+  && apt install --yes --no-install-recommends build-essential libjpeg-dev libpng-dev pkg-config \
+  && npm ci \
+  && npm link \
+  && npm cache clean --force \
+  && apt purge --yes build-essential pkg-config \
```

Comment:
Remove packages that were only needed to build some dependencies from the source code.

This will free up 164 Mb.